support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -171,7 +171,7 @@ export function ContributionsOrderSummary({
 	onCheckListToggle,
 	headerButton,
 	tsAndCs,
-	startDateTierThree,
+	startDate,
```

Comment:
Not related to this PR but we should probably be using this for newspaper as well as that also has a start date.

Reply:
Yes I wondered whether worth doing that.